Cython/Includes/cpython/pyport.pxd

```diff
@@ -0,0 +1,8 @@
+cdef extern from "Python.h":
+    ctypedef int int32_t
+    ctypedef int int64_t
+    ctypedef unsigned int uint32_t
+    ctypedef unsigned int uint64_t
```

Comment:
my intuition is that int32_t and int64_t should mean roughly what they mean when cimported from numpy, but they're the same here.  what am i missing?